cloud-agent/service/server/src/main/resources/application.conf

```diff
@@ -52,6 +52,8 @@ pollux {
   syncRevocationStatusesBgJobProcessingParallelism = ${?SYNC_REVOCATION_STATUSES_BG_JOB_PROCESSING_PARALLELISM}
   credential.sdJwt.expiry = 30 days
   credential.sdJwt.expiry = ${?CREDENTIAL_SD_JWT_EXPIRY}
+  presentationInvitationExpiry = 300 seconds
+  presentationInvitationExpiry = ${?PRESENTATION_INVITATION_EXPIRY}
```

Comment:
@mineme0110, don't forget to add a new environment variable to the [documentation](https://hyperledger.github.io/identus-docs/docs/identus/cloud-agent/environment-variables)

Reply:
@yshyn-iohk  Yes I have ticket for documentation I will remember about this